Lib/dis.py

```diff
@@ -191,8 +191,32 @@ def show_code(co, *, file=None):
     """
     print(code_info(co), file=file)
 
-_Instruction = collections.namedtuple("_Instruction",
-     "opname opcode arg argval argrepr offset starts_line is_jump_target")
+Positions = collections.namedtuple(
```

Comment:
Shoudln't this be `_Positions` the same way the other one is `_Instruction`?

Reply:
Why? We don't need to extend the actual class with other methods (unlike in the case of `Instruction`).

Reply:
:+1: